js/faq.js

```diff
@@ -1,8 +1,34 @@
 const faqLinks = document.querySelectorAll('.faq_link');
 const currentLocation = window.location.hash;
 
+window.addEventListener('keydown', (e) => {
```

Comment:
can we have a single-line comment above this function regarding what the purpose of this function is? It will help developers to understand the purpose of this function on a high level

Reply:
done